src/dev/code_coverage/ingest_coverage/integration_tests/ingest_coverage.test.js

```diff
@@ -0,0 +1,181 @@
+
+        beforeAll(done => {
+          const ingestAndMutateAsync = ingestAndMutate(done);
+          const ingestAndMutateAsyncWithPath = ingestAndMutateAsync(bothIndexesPath);
+          const verboseIngestAndMutateAsyncWithPath = ingestAndMutateAsyncWithPath(verboseArgs);
+          verboseIngestAndMutateAsyncWithPath(mutableBothIndexesChunks);
+        });
+
+        it(
+          'should result in every posted item having a site url that meets all regex assertions',
+          always(
+            siteUrlsSplitByNewLineWithoutBlanks(mutableBothIndexesChunks).forEach(
+              expectAllRegexesToPass(staticSiteUrlRegexes)
+            )
+          )
+        );
+
+        it('should result in the "just logging" message being present in the log', () => {
+          expect(mutableBothIndexesChunks.some(x => x.includes('Just Logging'))).to.be(true);
+        });
+        it('should result in the "actually sending" message NOT being present in the log', () => {
+          expect(mutableBothIndexesChunks.every(x => !x.includes('Actually sending...'))).to.be(
+            true
+          );
+        });
+        describe(`with provided vcs info file`, () => {
+          const filterZero = xs => included => xs.filter(x => x.includes(included))[0];
+          const filteredWith = filterZero(mutableBothIndexesChunks);
+          it('should have a vcs block', () => {
+            const vcs = 'vcs';
+            const portion = filteredWith(vcs);
+            expect(portion).to.contain(vcs);
+          });
+          it(`should have a branch`, () => {
+            const branch = `"branch":`;
+            const portion = filteredWith(branch);
+            expect(portion).to.contain(branch);
+            expect(portion).to.contain(`"origin/ingest-code-coverage"`);
+          });
+          it(`should have a sha`, () => {
+            const sha = `"sha":`;
+            const portion = filteredWith(sha);
+            expect(portion).to.contain(sha);
+            expect(portion).to.contain(`"f07b34f6206"`);
+          });
+          it(`should have an author`, () => {
+            const author = `"author":`;
+            const portion = filteredWith(author);
+            expect(portion).to.contain(author);
+            expect(portion).to.contain(`"Tre' Seymour"`);
+          });
+          it(`should have a commit msg, truncated`, () => {
+            const commitMsg = `"commitMsg":`;
+            const portion = filteredWith(commitMsg);
+            expect(portion).to.contain(commitMsg);
+            expect(portion).to.contain(`"Lorem :) ipsum Tre' λ dolor sit amet, consectetur ..."`);
+          });
+        });
+      });
+    });
+  });
+});
+
+function ingestAndMutate(done) {
+  return summaryPathSuffix => args => xs => {
+    const coverageSummaryPath = resolve(MOCKS_DIR, summaryPathSuffix);
+    const opts = [...args, coverageSummaryPath];
+    const ingest = spawn(process.execPath, opts, { cwd: ROOT_DIR, env });
```

Comment:
Are we actually ingesting these into the `kibana-coverage.elastic.dev` as part of the integration tests?

Reply:
No sir, there's an env flag